app/components/VideoComponents/VideoSet.js

```diff
@@ -167,7 +181,32 @@ export default class VideoSet extends Component<Props, State> {
     const { classifierType, rawEEGObservable } = this.props.location.state;
     const classifierPipe =
       classifierType === 'alpha' ? computeAlpha : computeThetaBeta;
-    // start recording raw EEG
+
+    // TODO: this worskpacedir variable needs to be set by a workspace property inherited from Home, ideally unique for each
+    const workspaceDir = createWorkspaceDir(
+      this.props.location.state.subjectId
```

Comment:
If this function is not working, let me know what the error message you're getting is. It's what I use in BrainWaves as a cross-platform way to create a directory on the file system

Reply:
Hard for me to tell is this function isn't working because I used the above and the following
`const workspaceDir = 'RAW EEG';`

And both definitions gave me the following error:
![screenshot 70](https://user-images.githubusercontent.com/3487193/52983850-f3479880-33ba-11e9-916f-58542cd9f421.png)

Let me know if there's something else I can do to help you troubleshoot.

Reply:
Looks I accidentally typed 'timestamp' instead of 'timestamps' in the new timestamping code in the raw eeg observable. I've made the change